docs/sections/user_guide/cli/mode_template.rst

```diff
@@ -24,54 +24,57 @@ The ``uw`` mode for handling :jinja2:`Jinja2 templates<templates>`.
 ``render``
 ----------
 
-.. code:: sh
+.. code-block:: text
 
   $ uw template render --help
-  usage: uw template render [-h] [--input-file PATH] [--output-file PATH] [--values-file PATH] [--values-format {ini,nml,sh,yaml}] [--values-needed] [--dry-run] [--quiet] [--verbose] [KEY=VALUE ...]
+  usage: uw template render [-h] [--input-file PATH] [--output-file PATH] [--values-file PATH]
+                            [--values-format {ini,nml,sh,yaml}] [--values-needed] [--dry-run]
+                            [--quiet] [--verbose]
+                            [KEY=VALUE ...]
 
   Render a template
 
   Optional arguments:
-   -h, --help
-         Show help and exit
-   --input-file PATH, -i PATH
-         Path to input file (defaults to stdin)
-   --output-file PATH, -o PATH
-         Path to output file (defaults to stdout)
-   --values-file PATH
-         Path to file providing override or interpolation values
-   --values-format {ini,nml,sh,yaml}
-         Values format
-   --values-needed
-         Print report of values needed to render template
-   --dry-run
-         Only log info, making no changes
-   KEY=VALUE
-         A key=value pair to override/supplement config
-   --quiet, -q
-         Print no logging messages
-   --verbose, -v
-         Print all logging messages
+    -h, --help
+        Show help and exit
+    --input-file PATH, -i PATH
+        Path to input file (defaults to stdin)
+    --output-file PATH, -o PATH
+        Path to output file (defaults to stdout)
+    --values-file PATH
+        Path to file providing override or interpolation values
+    --values-format {ini,nml,sh,yaml}
+        Values format
+    --values-needed
+        Print report of values needed to render template
+    --dry-run
+        Only log info, making no changes
+    --quiet, -q
+        Print no logging messages
+    --verbose, -v
+        Print all logging messages
+    KEY=VALUE
+        A key=value pair to override/supplement config
```

Comment:
An update from `develop` leads to the positional `KEY=VALUE` args now appearing last in the help output.